core/rfb.js

```diff
@@ -369,7 +369,11 @@
                 uri = this._encrypt ? 'wss' : 'ws';
             }
 
-            uri += '://' + this._rfb_host + ':' + this._rfb_port + '/' + this._rfb_path;
+            uri += '://' + this._rfb_host;
+            if(this._rfb_port)
+                uri += ':' + this._rfb_port;
```

Comment:
Sorry, I can fix this up.  I had also planned on adjusting the test case so that the build checks passed.